packages/verification/src/rest/v1/verifications/verifications-api.ts

```diff
@@ -211,10 +266,59 @@ export class VerificationsApi extends VerificationDomainApi {
     });
   }
 
+  /**
+   * Report a Phone Call verification using Identity
+   * Report the received verification code (OTP) to verify it, using the identity of the user (in most cases, the phone number).
+   * @param { ReportPhoneCallVerificationByIdentityRequestData } data - The data to provide to the API call.
+   */
+  public async reportPhoneCallByIdentity(
+    data: ReportPhoneCallVerificationByIdentityRequestData,
+  ): Promise<PhoneCallVerificationReportResponse> {
+    this.client = this.getSinchClient();
+    (data.reportPhoneCallVerificationByIdentityRequestBody as any).method = 'callout';
+    const getParams = this.client.extractQueryParams<ReportPhoneCallVerificationByIdentityRequestData>(
+      data, [] as never[]);
+    const headers: { [key: string]: string | undefined } = {
+      'Content-Type': 'application/json; charset=UTF-8',
```

Comment:
outside of the PR scope, but:
1. Verification OAS file do not specify any charset, only 'application/json' and is compliant with OAS (https://spec.openapis.org/oas/v3.0.3.html#fixed-fields-10)
2. 'application/json' by default mean UTF-8 (RFC chapter  "3. Encoding": https://www.ietf.org/rfc/rfc4627.txt)

Reply:
Indeed, there has been a spec update from the time these files were generated.